front/src/common/Map/Layers/extensions/SNCF/PSLSigns.tsx

```diff
@@ -13,6 +13,7 @@ import type { LayerContext } from 'common/Map/Layers/types';
 import getKPLabelLayerProps from 'common/Map/Layers/KPLabel';
 
 import { getMap } from 'reducers/map/selectors';
+import { getCommonSignsMastLayerProps } from '../../MastsLayers';
```

Comment:
No relative paths please.
```suggestion
import { getCommonSignsMastLayerProps } from 'common/Map/Layers/MastsLayers';
```

Reply:
changed to absolute paths